cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -973,11 +1036,11 @@ func init() {
 	endDevicesClaimCommand.Flags().String("target-application-server-id", "", "")
 	endDevicesClaimCommand.Flags().Bool("invalidate-authentication-code", true, "invalidate the claim authentication code to block subsequent claiming attempts")
 	endDevicesCommand.AddCommand(endDevicesClaimCommand)
+	endDevicesCommand.AddCommand(endDevicesListQRCodeFormatsCommand)
 	endDevicesGenerateQRCommand.Flags().AddFlagSet(endDeviceIDFlags())
-	endDevicesGenerateQRCommand.Flags().String("format", "tr005draft2", "format (tr005draft2)")
-	endDevicesGenerateQRCommand.Flags().Int("size", 300, "")
-	endDevicesGenerateQRCommand.Flags().String("file", "", "file to write QR code in PNG format")
-	endDevicesGenerateQRCommand.Flags().Bool("print", false, "print QR code as text")
+	endDevicesGenerateQRCommand.Flags().String("format-id", "tr005draft2", "")
```

Comment:
If we set this as a default, we'll be stuck with it until v3, otherwise we would break CLI compatibility